System.Security.Cryptography/Properties/AssemblyInfo.cs

```diff
@@ -16,5 +16,5 @@
 
 ////////////////////////////////////////////////////////////////
 // update this whenever the native assembly signature changes //
-[assembly: AssemblyNativeVersion("100.0.0.2")]
+[assembly: AssemblyNativeVersion("100.0.0.3")]
```

Comment:
That means, you'll have to raise a PR on the native side as well to keep it in sync

Reply:
Yes, it's already there nanoframework/nf-interpreter#2888